src/configure/mod.rs

```diff
@@ -517,6 +533,51 @@ async fn try_smtp_one_param(
     }
 }
 
+async fn add_all_receipients_as_contacts(
+    ctx: &Context,
+    imap: &mut Imap,
+    folder: Config,
+) -> Option<()> {
```

Comment:
this seems to be a very odd return type, what are you trying to express?

Reply:
I wanted to be able to use the `?` operator but I can change this

Reply:
I changed it but it's such a pity that I can't return in unwrap_or_else(), Rust does not have an Elvis operator and you can't use the ? operator in fn's that return nothing.